lib/ExpressRedisCache/route.js

```diff
@@ -166,7 +166,11 @@ module.exports = (function () {
           /** if it's cached, display cache **/
 
           if ( cache.length ) {
-            res.contentType(cache[0].type || "text/html");
+            if (res.contentType) {
+				      res.contentType(cache[0].type || "text/html");
+			      } else if (res.setHeader) {
+				      res.setHeader('content-type', cache[0].type || "text/html");
+			      }
```

Comment:
Looks like spaces are being used elsewhere in this project. Would you consider uses spaces instead of tabs for consistency?